### DISCUSSION TITLE
Is there an option to also delete the namespace on "helmfile destroy"?

### DISCUSSION THREAD

**Discussion by tibeer** [helmfile/helmfile#1379] [Questions]
I usually create one namespace per helm release. As helmfile allows me to use helm in a declarative way, I can create namespaces on "helmfile apply". But can I also delete a namespace when using "destroy"?

**Comment by yxxhero** [comment]
@tibeer The answer is no. it only delete resource that controled by helm. if you want to delete namespace, you can add the `namespace` yaml to the helm template.

**Reply by tibeer** [reply]
Awesome, thanks for the quick reply 🚀
